plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchExecutor.groovy

```diff
@@ -217,7 +219,7 @@ class AwsBatchExecutor extends Executor implements ExtensionPoint {
      */
     private ThrottlingExecutor createExecutorService(String name) {
 
-        final qs = session.getQueueSize(name, 5_000)
+        final qs = session.config.navigate("threadPool.${name}.maxQueueSize",5_000) as Integer
```

Comment:
So you want to just do `final qs = 5_000` and let it be overwritten by the submitter options?